pkg/report/sarif.go

```diff
@@ -0,0 +1,61 @@
+package report
+
+import (
+	"github.com/owenrumney/go-sarif/sarif"
+	"io"
+)
+
+// SarifWriter implements result Writer
+type SarifWriter struct {
+	Output io.Writer
+}
+
+func (sw SarifWriter) Write(report Report) error {
+	sarifReport, err := sarif.New(sarif.Version210)
+	if err != nil {
+		return err
+	}
+	run := sarif.NewRun("Trivy", "https://github.com/aquasecurity/trivy")
+	run.Tool.Driver.WithVersion("0.15.0")
```

Comment:
Looks like we have to pass the tool version.
https://github.com/owenrumney/go-sarif/blob/91aa28321332bab3e989f2446fbcb6dcf80ae95e/sarif/tool.go#L16

Reply:
ok, sure